.env

```diff
@@ -40,7 +40,7 @@ DATABASE_URL=mysql://db:db@mariadb:3306/db
 CONFIGURATION_URL=APP_CONFIGURATION_URL
 CLIENT_ID=APP_CLIENT_ID
 CLIENT_SECRET=APP_CLIENT_SECRET
-CALLBACK_URI=APP_CALLBACK_URI
+REDIRECT_URI=APP_REDIRECT_URI
```

Comment:
this is mentioned in the readme, so should be updated

Reply:
updated in the readme that is

Reply:
Fixed